shared/src/mobileMain/kotlin/dev/johnoreilly/confetti/SearchViewModel.kt

```diff
@@ -0,0 +1,96 @@
+package dev.johnoreilly.confetti
+
+import com.rickclephas.kmm.viewmodel.KMMViewModel
+import dev.johnoreilly.confetti.fragment.SessionDetails
+import dev.johnoreilly.confetti.fragment.SpeakerDetails
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.filterIsInstance
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.update
+
+class SearchViewModel(
+    // TODO: Remove dependency between view models.
+    private val sessionsViewModel: SessionsViewModel,
+    private val speakersViewModel: SpeakersViewModel,
```

Comment:
You can use `ConfettiRepository.conferenceData()` and `ConfettiRepository.bookmarks()` that should contain all this information.